src/typescript/05-modules/modules.md

```diff
@@ -84,26 +84,14 @@ const constrained = constrain(-18, 0, 100);
 
 In the code above, we can import the `constrain` function by name from `helpers.ts` and use it in our code.
 
-#### Exporting in place
-
-You can also export values from where they’re declared in the file, without a separate `export` statement:
-
-```ts
-export function constrain(value: number, min: number, max: number): number {
-  return Math.min(Math.max(value, min), max);
-}
-```
-
-@highlight 1
```

Comment:
this isn't needed, as the example above now does exporting in place, from notes:

![Screenshot 2024-05-20 at 1 00 31 PM](https://github.com/bitovi/academy/assets/9858612/23340dfd-9514-4e4e-b95d-fc7a9d337940)